### DISCUSSION TITLE
3rd party screenshot apps

### DISCUSSION THREAD

**Discussion by AdriSonet** [sumatrapdfreader/sumatrapdf#3451] [Ideas]
Good morning, 

My suggestion would be that when you're using 3rd party apps that auto-detect windows, forms.. when taking a screenshot (I'm talking about ShareX in this case) it would be interesting to see an auto-adjust to the page of the pdf itself for easy screenshooting. If i didn't explain correctly, there's a screenshot 

https://i.imgur.com/LhaC6lb.png

I'm not sure this is a thing that can be done within Sumatra. 

Thank you

**Comment by GitHubRulesOK** [comment]
Windows screen shot can be area or window or screen has always been whichever  I need at the time. say for screens like this page

![image](https://user-images.githubusercontent.com/12754913/233394756-835005a9-8664-466f-be9b-cb79a3dcf4d4.png)

For inside SumatraPDF canvas select single page and set the pixel size you want say 200% then copy page (CTRL+A) or an area like this and paste the double quality image in MSPaint Thats the same as I used in Acrobat almost 30 years ago, and still works with the same quality as then.

![image](https://user-images.githubusercontent.com/12754913/233396282-98cc75c8-036f-4c9b-a7be-fce9fd4d21a0.png)

output
---
![image](https://user-images.githubusercontent.com/12754913/233396643-23c06deb-1400-454d-b936-350a2bf3c9db.png)
